pkg/volume/volume.go

```diff
@@ -122,27 +122,29 @@ func Describe(vols []string, openebsNs string) error {
 // CasList returns a list of functions by cas-types for volume listing
 func CasList() []func(*client.K8sClient, *corev1.PersistentVolumeList, string) ([]metav1.TableRow, error) {
 	// a good hack to implement immutable lists in Golang & also write tests for it
-	return []func(*client.K8sClient, *corev1.PersistentVolumeList, string) ([]metav1.TableRow, error){GetJiva, GetCStor, GetZFSLocalPVs, GetLVMLocalPV}
+	return []func(*client.K8sClient, *corev1.PersistentVolumeList, string) ([]metav1.TableRow, error){GetJiva, GetCStor, GetZFSLocalPVs, GetLVMLocalPV, GetLocalHostpath}
 }
 
 // CasListMap returns a map cas-types to functions for volume listing
 func CasListMap() map[string]func(*client.K8sClient, *corev1.PersistentVolumeList, string) ([]metav1.TableRow, error) {
 	// a good hack to implement immutable maps in Golang & also write tests for it
 	return map[string]func(*client.K8sClient, *corev1.PersistentVolumeList, string) ([]metav1.TableRow, error){
-		util.JivaCasType:  GetJiva,
-		util.CstorCasType: GetCStor,
-		util.ZFSCasType:   GetZFSLocalPVs,
-		util.LVMCasType:   GetLVMLocalPV,
+		util.JivaCasType:   GetJiva,
+		util.CstorCasType:  GetCStor,
+		util.ZFSCasType:    GetZFSLocalPVs,
+		util.LVMCasType:    GetLVMLocalPV,
+		util.LocalHostpath: GetLocalHostpath,
 	}
 }
 
 // CasDescribeMap returns a map cas-types to functions for volume describing
 func CasDescribeMap() map[string]func(*client.K8sClient, *corev1.PersistentVolume) error {
 	// a good hack to implement immutable maps in Golang & also write tests for it
 	return map[string]func(*client.K8sClient, *corev1.PersistentVolume) error{
-		util.JivaCasType:  DescribeJivaVolume,
-		util.CstorCasType: DescribeCstorVolume,
-		util.ZFSCasType:   DescribeZFSLocalPVs,
-		util.LVMCasType:   DescribeLVMLocalPVs,
+		util.JivaCasType:   DescribeJivaVolume,
```

Comment:
This code is not gofmted

Reply:
This was formated by the golang, since there was new addition with slightly bigger name hence, these are shifted a bit ahead.